corehq/apps/cloudcare/views.py

```diff
@@ -264,8 +263,7 @@ def get_cases(request, domain):
     ids_only = string_to_boolean(request.REQUEST.get("ids_only", "false"))
     case_id = request.REQUEST.get("case_id", "")
     footprint = string_to_boolean(request.REQUEST.get("footprint", "false"))
-    include_children = string_to_boolean(request.REQUEST.get("include_children", "false"))
-    if case_id and not footprint and not include_children:
+    if case_id and not footprint:
```

Comment:
@gcapalbo git blame led me here: https://github.com/dimagi/commcare-hq/pull/621/ - is that fix/change no longer necessary?